### PR TITLE
Corner sides case sensitive fix

### PR DESCRIPTION
**Pull request by mmadjer**
Make corner sides case insensitive.
See issue #394.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ None
 [#368](https://github.com/IBAnimatable/IBAnimatable/pull/403)
 
 #### Bugfixes
-None
+- Make corner sides case insensitive.
+[#394](https://github.com/IBAnimatable/IBAnimatable/issues/394)
```
Comment (JakeLin) [inline CHANGELOG.md]: Could you please add your name here 😊

File: IBAnimatable/CornerSide.swift [hunk 2]
```diff
@@ -57,10 +57,10 @@ public struct CornerSides: OptionSet {
     }
 
     switch side {
-    case .topLeft: self = .topLeft
-    case .topRight: self = .topRight
-    case .bottomLeft: self = .bottomLeft
-    case .bottomRight: self = .bottomRight
+    case .topleft: self = .topleft
```
Comment (JakeLin) [inline IBAnimatable/CornerSide.swift:60]: can we do `self = side` here without the `switch`?

**Review by JakeLin — APPROVED** [review]
Thanks, it is ready to go 👍

File: IBAnimatable/CornerSide.swift (hunk 2 above)
Comment (mmadjer) [inline IBAnimatable/CornerSide.swift]: good catch. will change it.

Comment (mmadjer) [reply IBAnimatable/CornerSide.swift:60]: no we can't, as it maps to the CornerSides option types.

**Review by tbaranes — APPROVED** [review]

File: IBAnimatable/CornerSide.swift (hunk 2 above)
Comment (JakeLin) [reply IBAnimatable/CornerSide.swift:60]: @mmadjer Good found, I got confused 😕

**Comment by JakeLin** [comment]
@mmadjer thanks 👍👍👍

**Comment by tbaranes** [comment]
Second thought: are we introducing a breaking changes by changing the `CornerSide` rawValue ?

**Comment by JakeLin** [comment]
@tbaranes I think we did  😢，good pick-up. It will be no problem when we use it in IB. But need to change when we use it in code. Since we have merged it already, I am going to update the CHANGELOG.md now.

Update: commit 2f05c225345f4452f3c3245e32c374ac238cdcb4 pushed to describe the breaking change.

**Comment by tbaranes** [comment]
Can't we just fix it by giving a value to the enum?

```
case topLeft: "topleft"
...
```

**Comment by mmadjer** [comment]
Ah sh**, good thinking @tbaranes.

**Comment by JakeLin** [comment]
@tbaranes well, such a good idea. 

@mmadjer Could you please give it a go if you have time?

**Pull request merged**